frontend/src/interface/PdfExporter.js

```diff
@@ -0,0 +1,172 @@
+import { jsPDF } from 'jspdf';
+import html2canvas from 'html2canvas';
+import axios from 'axios';
+import FirstPDFPage from 'assets/pdfImages/firstPage.jpg';
+import SecondPDFPage from 'assets/pdfImages/secondPage.jpg';
+import ThirdPDFPage from 'assets/pdfImages/thirdPage.jpg';
+import FourthPDFPage from 'assets/pdfImages/fourthPage.jpg';
+import FifthPDFPage1 from 'assets/pdfImages/fifthPage1.jpg';
+import FifthPDFPage2 from 'assets/pdfImages/fifthPage2.jpg';
+import SixthPDFPage from 'assets/pdfImages/sixthPage.jpg';
+
+/**
+ * generating pdf for the baseline scenario
+ *
+ * @author Parham Gandomkar
```

Comment:
I believe we are in the process of removing author tags. Because in Scrum everyone should be responsible, and git blame can tell you very precise who worked on the code you are looking at the last. 

You can install GitLens Exatention for Visual Studio Code to see authors always. 🙂 @gandompm